resources/public/api/conf/1.0/schemas/retrieve_relief_foreign_response.json

```diff
@@ -53,6 +53,7 @@
         }
       },
       "required": [
```

Comment:
this required section is in the wrong place, so they arn't showing as being required when looking at the raml
needs to be one bracket higher.